src/forms/SelectMultipleFilterPlainForm/SelectMultipleFilterPlainForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { arrayOf, bool, node, shape, string, func } from 'prop-types';
+import { Form as FinalForm, FormSpy } from 'react-final-form';
+import arrayMutators from 'final-form-arrays';
+
+import { FieldCheckboxGroup, Form } from '../../components';
+
+const SelectMultipleFilterPlainForm = props => {
+  const { onChange, ...rest } = props;
+
+  const handleChange = formState => {
+    if (formState.dirty) {
+      onChange(formState.values);
+    }
+  };
+
+  return (
+    <FinalForm
+      {...rest}
+      onSubmit={() => null}
+      mutators={{ ...arrayMutators }}
+      render={({ className, name, options, twoColumns, onChange }) => (
```

Comment:
We haven't used this spread format of props with other components. It helps to debug code if there's full prop list available.